logging/logger.go

```diff
@@ -10,6 +10,7 @@ type Logger interface {
 	ZapCompatibleLogger
 
 	SetLevel(level Level)
+	GetLevel() Level
```

Comment:
Not strictly necessary (`ZapCompatibleLogger` has a `Level() zapcore.Level`). But I had some printlns and the mixing of cases in `zapcore.Level.String` and `logging.Level.String` bothered me.